packages/perspective-jupyterlab/src/less/index.less

```diff
@@ -1,7 +1,7 @@
-@import "@finos/perspective-viewer/src/less/fonts.less";
-@import "@finos/perspective-viewer/src/themes/material-dense.less";
+// @import "../../../../rust/perspective-viewer/src/less/fonts.less";
+@import "../../../../rust/perspective-viewer/src/themes/material-dense.less";
```

Comment:
Can you symlink into node_modules in `postinstall` (like for workspace packages) and avoid a big relative import?

Reply:
These are relative because of the introduction of `externals` in package.json, e.g. they are ultimately-inlined shared code between packages, not runtime dependencies.  Docs for these changes is forthcoming.

Reply:
Removed this change and made `src` public.